sdk/communication/communication-phone-numbers/package.json

```diff
@@ -123,6 +121,14 @@
     "typedoc": "0.15.2"
   },
   "//sampleConfiguration": {
+    "productName": "Azure Communication Services",
```

Comment:
Since these samples are specifically for communication-phone-numbers, we should have some kind of indication of that here, however it makes sense or however you prefer to stylize it. Here are my initial thoughts:

- Azure Communication Services Phone Numbers
- Azure Communication Services - Phone Numbers

If phone numbers isn't included in the product name, then all the ACS samples would end up with the same title, since it's generated from this entry.